src/core/main.lua

```diff
@@ -142,6 +142,14 @@ end
 -- Cleanup after Snabb process.
 function shutdown (pid)
    if not _G.developer_debug and not lib.getenv("SNABB_SHM_KEEP") then
+      -- Check if there is a backlink to the named app, if so cleanup that.
+      local backlink = shm.root.."/"..pid.."/name"
+      if S.lstat(backlink) then
```

Comment:
Would you mind removing this if?  Instead if the readlink on the next line fails, you will get a nil name_link, and in that case you replace line 149 with if name_link then S.unlink(name_link) end, and you ignore errors in any case.